bin/alluxio-mount.sh

```diff
@@ -208,8 +208,8 @@ function mount_local() {
       DECL_INIT=$(declare -f init_env)
       DECL_MEM_SIZE_TO_BYTES=$(declare -f mem_size_to_bytes)
       DECL_MOUNT_LINUX=$(declare -f mount_ramfs_linux)
-      sudo bash -O extglob -c "BIN=${BIN}; ${DECL_INIT}; ${DECL_MEM_SIZE_TO_BYTES}; \
```

Comment:
Need to pass `ALLUXIO_CONF_DIR` to `mount_ramfs_linux`, which will source `${ALLUXIO_CONF_DIR}/alluxio-env.sh`